src/tree_schema_compile.c

```diff
@@ -6187,8 +6248,18 @@ lys_schema_node_get_module(const struct ly_ctx *ctx, const char *nametest, size_
     return target_mod;
 }
 
+/**
+ * @brief Check whether a parsed node matches a single schema nodeid name test.
+ *
+ * @param[in] node_p Parsed node to consider.
+ * @param[in] node_p_mod Compiled @p node_p to-be module.
+ * @param[in] mod Expected module.
+ * @param[in] name Expected name.
+ * @param[in] name_len Length of @p name.
+ * @return Whether it is a match or not.
+ */
 static ly_bool
-lysp_schema_nodeid_match_node_p(const struct lys_module *node_p_mod, const struct lysp_node *node_p,
+lysp_schema_nodeid_match_node_p(const struct lysp_node *node_p, const struct lys_module *node_p_mod,
```

Comment:
More a generic note - seems that we are using the `_p` suffix in 2 different meanings. Here it is as _parsed_, but at other places it is kind of flag for return pointer. Probably we should unify its usage to avoid confusion.

Reply:
Fixed by renaming `node_p` to `pnode`.